package/Dockerfile

```diff
@@ -1,5 +1,4 @@
-FROM gcr.io/distroless/static
+FROM registry.suse.com/bci/bci-base:15.3
+RUN zypper -n install pciutils
 COPY bin/pcidevices /bin/pcidevices
 ENTRYPOINT ["pcidevices"]
-# https://pci-ids.ucw.cz/v2.2/pci.ids mirror
-ADD https://github.com/pciutils/pciids/raw/master/pci.ids /usr/share/hwdata/pci.ids
```

Comment:
If I may suggest updating the PCI ID list (`update-pciids`).

Reply:
Good call, pushing up a new revision in a moment